Taskfile.yml

```diff
@@ -109,6 +109,22 @@ tasks:
     cmds:
       - yarn wsrun --report --fast-exit --exclude-missing ws:clean
 
+  workspaces:watch:
+    desc: Watch source files and build as they change
+    deps: [workspaces:watch:typescript]
+    cmds:
+      - echo "Watching ended"
+
+  workspaces:watch:typescript:
+    desc: Watch all the source of Typescripts workspaces and automatically recompile (with a single watcher)
+    cmds:
+      - yarn tsc -b --watch
+
+  workspaces:watch:wasm:
+    desc: Watch the source of WASM targets and automatically recompile
+    cmds:
+      - yarn wsrun --parallel --exclude-missing ws:watch-wasm
+
```

Comment:
@notnmeyer this is the best way I could imagine doing watchers with Taskfile. Do you see any problems with this / know of a better way?

Reply:
sorry, been out. this looks fine to me though. i dont think its a better solution in this case, but taskfile also supports fingerprinting source files and watching a task too,

https://github.com/go-task/task/blob/master/docs/usage.md#watch-tasks
https://github.com/go-task/task/blob/master/docs/usage.md#by-fingerprinting-locally-generated-files-and-their-sources